pyuvdata/tests/test_uvdata.py

```diff
@@ -234,48 +227,46 @@ def teardown(self):
 
     def test_baseline_to_antnums(self):
         """Test baseline to antnum conversion for 256 & larger conventions."""
-        nt.assert_equal(self.uv_object.baseline_to_antnums(67585), (0, 0))
-        nt.assert_raises(
-            Exception, self.uv_object2.baseline_to_antnums, 67585)
+        assert self.uv_object.baseline_to_antnums(67585) == (0, 0)
+        pytest.raises(Exception, self.uv_object2.baseline_to_antnums, 67585)
```

Comment:
Should this be a more specific Exception type?

Reply:
It should be, I'll dig in and check the exact Exception raised

Reply:
this tests the following lines https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/63d6f7f7bb30ddea01f917b9a031a03ff52f20a5/pyuvdata/utils.py#L936-L939

Reply:
These lines also raise just an `Exception`
https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/63d6f7f7bb30ddea01f917b9a031a03ff52f20a5/pyuvdata/utils.py#L908-L910

Reply:
hmm, interesting. Thanks for digging in!